### PR TITLE
Single-cell RNA-Seq: Tutorial "Clustering 3K PBMCs with Scanpy"

### PR DESCRIPTION
**Pull request by bebatut**
Still need to do:
- [x] Add questions
- [x] Add objectives
- [x] Add take-home messages

But the content of the tutorial itself is ready for review (@mtekman and any single-cell person) and to be tested

### CODE REVIEW

File: topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,1814 @@
...
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Data
+
+For this tutorial, we analyze a dataset of Peripheral Blood Mononuclear Cells (PBMC) extracted from a heal, freely available from 10X Genomics. The dataset contains 2,700 single cells sequencd using Illumina NextSeq 500. The raw sequences have been processed by [**cellranger**](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger) pipeline from 10X to extract an unique molecular identified (UMI) count matrix, in a similar way as explained in ["Pre-processing of 10X Single-Cell RNA Datasets" tutorial]({{ site.baseurl }}{% link topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.md %}).
```
Comment (mtekman) [inline topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md]: heal → healthy donor

**Comment by mtekman** [comment]
this looks amazing, give me an hour or two to make some small wording changes

**Comment by bgruening** [comment]
@mtekman can you please use the suggestion mode in github or push to this branch.
@hrhotz thanks for the review!

And of course, @bebatut amazing work. Enjoy your days off!

**Comment by mtekman** [comment]
@bgruening Yep I will push

**Comment by mtekman** [comment]
@bebatut The tutorial looks great from my side - each section follows nicely from the other, there are a lot of details given without ever losing context, and I feel that the user becomes very intimate with the AnnData format structure so that they know exactly what is being added and manipulated at each step. The explanation of PC's was excellent and I think you really covered everything. Kudos!

**Comment by bebatut** [comment]
Thanks a lot for the reviews and suggested changes! 😄

**Pull request merged**